WebDriverAgentLib/Utilities/FBConfiguration.h

```diff
@@ -19,6 +19,9 @@ NS_ASSUME_NONNULL_BEGIN
 /*! If set to YES will ask TestManagerDaemon for element visibility */
 @property (class, nonatomic, assign) BOOL shouldUseTestManagerForVisibilityDetection;
 
+/* The maximum typing frequency for all typing activities */
```

Comment:
That's what I'm assuming. All I really know is that 60 is too high in many cases.